plugins/modules/clustering/pacemaker/pacemaker_cluster.py

```diff
@@ -0,0 +1,369 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2016, Mathieu Bultel <mbultel@redhat.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: pacemaker_cluster
+short_description: Manage pacemaker clusters
+author:
+- Mathieu Bultel (@mbultel)
+- Hani Audah (@haudah)
+description:
+   - This module can manage a pacemaker cluster and nodes from Ansible using
+     the pacemaker cli.
+options:
+    state:
+      description:
+        - Indicate desired state of the cluster
```

Comment:
```suggestion
        - Indicate desired state of the cluster.
```
Same for most other descriptions.